packages/kubernetes-1.26/kubelet-exec-start-conf

```diff
@@ -2,11 +2,7 @@
 ExecStart=
 ExecStart=/usr/bin/kubelet \
 {{#unless settings.kubernetes.standalone-mode}}
-{{#if (eq settings.kubernetes.cloud-provider "\"\"")}}
-    --cloud-provider "" \
-{{else}}
-    --cloud-provider "external" \
-{{/if}}
+    --cloud-provider {{default "external" settings.kubernetes.cloud-provider}} \
```

Comment:
What will be the value of `--cloud-provider` if `settings.kubernetes.cloud-provider` is `""`? Will it be `external` or will the handlebars helper respect the empty value?

Reply:
The handlebars will respect the `""` value (which is non-empty). This is the same behavior as all prior packaged K8s versions as well.

Reply:
See https://github.com/bottlerocket-os/bottlerocket/blob/92b93e02f7b9cf0344f3f835762b657aa88cb90f/sources/models/src/modeled_types/kubernetes.rs#L751-L752